src/test/java/com/redislabs/redisgraph/RedisGraphAPITest.java

```diff
@@ -765,7 +765,9 @@ public void testReadTransactionWatch(){
         c1.watch("social");
         RedisGraphTransaction t1 = c1.multi();
 
-        t1.query("social", "CREATE (:Person {name:'b'})");
+        Map<String, Object> params = new HashMap<>();
+        params.put("name", 'b');
+        t1.query("social", "CREATE (:Person {name:$name})", params);
```

Comment:
@DvirDukhan should you add a negative test too?

Reply:
there is one already